src/matlab2tikz.m

```diff
@@ -3710,6 +3710,9 @@ function warnFacetedInterp(m2t)
     str = sprintf('\\addplot[%s] plot table[%s] {%s};\n', ...
                  opts_print(m2t, drawOptions, ','), ...
                  opts_print(m2t, tabOpts, ','), table);
+
+    [m2t, baseline] = drawBaseline(m2t,h,isHoriz);
```

Comment:
Yes, you are correct. I was testing if you were paying attention :laughing:!
I will add comments, that I usually do at the end unless the code is hard to understand.